src/embed.js

```diff
@@ -65,34 +65,64 @@ function embed(el, opt, callback) {
   }
 
   vg.parse.spec(spec, function(chart) {
+    var renderer = opt.renderer || 'canvas',
+        isCanvas = renderer === 'canvas',
+        actions  = opt.actions || {};
```

Comment:
If `opt.actions === false` than this will set `actions = {}` and the subsequent if statement on L78 will not fail as intended.